### DISCUSSION TITLE
Is it possible to have github pages bandwidth increased or removed?

### DISCUSSION THREAD

**Discussion by vishal12312** [community/community#52458] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is it possible to have it changed to maybe 200GB or 1TB or have it removed all together? Thanks

**Comment by SMelidoni** [comment]
Hi @vishal12312 👋🏽

From what I've read, unfortunately I don't believe there is a built-in option to increase the bandwidth limit beyond the default. As of my knowledge the GitHub Pages usage limits are as follows:

- 100 GB of bandwidth per month
- 1 GB of storage

These limits apply to both free and paid GitHub accounts. If your project needs more bandwidth or storage, you may need to consider alternative hosting options, such as a dedicated web host, Content Delivery Network (CDN), or cloud-based storage services like AWS S3 or Google Cloud Storage.

You can read more about this by visiting the section *Usage Limits* in [About GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits) documentation.

Hope this helps!

**Comment by yoannchaudet** [comment]
Like said above we have service limits regulating bandwidth consumption. Those are specified so the service remain available to everyone. We do not monetize bandwidth consumption today as in, it is a free service. If your needs are going passed our service limits, you may need to find alternative hosting solutions.
